src/main/java/com/hivemq/client/internal/mqtt/message/publish/mqtt3/Mqtt3PublishView.java

```diff
@@ -50,8 +50,18 @@ public class Mqtt3PublishView implements Mqtt3Publish {
             final @NotNull MqttQos qos,
             final boolean retain) {
 
+        return delegate(topic, payload, qos, retain, null);
+    }
+
+    public static @NotNull MqttPublish delegate(
```

Comment:
this method should also not be duplicated

Reply:
Removed.